src/components/Gradebook/index.jsx

```diff
@@ -357,14 +357,14 @@ export default class Gradebook extends React.Component {
             <h1>Gradebook</h1>
             <h3> {this.props.courseId}</h3>
             {this.props.areGradesFrozen &&
-            <div className="alert alert-warning" role="alert" >
-              The grades for this course are now frozen. Editing of grades is no longer allowed.
-            </div>
+              <div className="alert alert-warning" role="alert" >
```

Comment:
I'm not sure why eslint flagged this for me all of a sudden.

Reply:
Hmmm... I suspect some version discrepancy nonsense is afoot here. We'll want to watch PRs for any edit warring on this point, and point folks to whatever gets decided on a canonicalized version a la https://edx-internal.slack.com/archives/CD4E4BCM9/p1564432657024900